build_library/disk_util

```diff
@@ -136,6 +136,8 @@ def LoadPartitionConfig(options):
       part.setdefault('fs_block_size', metadata['fs_block_size'])
       part.setdefault('fs_blocks', part['bytes'] // part['fs_block_size'])
       part['fs_bytes'] = part['fs_blocks'] * part['fs_block_size']
+      if part.get('extract_blocks', None):
```

Comment:
That'd be nice to have a comment here defining what is the `extract_blocks`. Sometimes it's used (`rootfs`), sometimes not (the rest)